commands/report.js

```diff
@@ -24,6 +24,7 @@ exports.run = async (client, message, args) => {
     return message.channel.send(`${redtick} Please specify an user and a reason`)
 
   const user = client.utils.parseMention(args.shift())
+  if(message.author.id === user) return message.reply("You shoudn't report yourself <:kappalul:527037429161525248>")
```

Comment:
Don't use double quotes, and please put the redtick emote on the response

Reply:
...done